bloom-housing/bloom
# feat: add address verification for preferences step

# Pull Request Template
## Issue Overview
This PR addresses #3672
## Description
Adds verification step for preferences that requires address verification. So the one that has validationMethod specified (so the one from geocoding pipeline)

## How Can This Be Tested/Reviewed?

All checkbox / radio questions in application that are not in preferences step should NOT be affected by this change. After populating database, without changes in settings => options it should also work the same as before.
in /settings => options you can set `Do you want to collect address information?` true and fill other options (doesn't really matter how for this change).
Then after checking a preference field that has address it should verify it after click `next`.
Try to check different multi checks variants (like first and last, first - middle - last, 2nd with address checked and then second last without address checked etc.).

## Checklist:
## Reviewer Notes:
Steps to review a PR:
- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go
## On Merge:
If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
